tools/triage_status_report.py

```diff
@@ -72,8 +72,7 @@ def triage_status_report(jira_client, filter_id, webhook):
     filter_url = jira_filter.viewUrl
 
     # Temporary focus
-    metal_4_11_multi = "https://issues.redhat.com/issues/?jql=project%20%3D%20AI-Triage%20AND%20component%20%3D%20Cloud-Triage%20AND%20created%20%3E%3D%20-7d%20AND%20labels%20%3D%20FEATURE-Platform-baremetal%20AND%20labels%20!%3D%20FEATURE-SNO%20AND%20affectedVersion%20%3D%20%22OpenShift%204.11%22%20ORDER%20BY%20key%20DESC"
-
+    ocp_4_12 = "https://issues.redhat.com/issues/?jql=project%20%3D%20AI-Triage%20AND%20component%20%3D%20Cloud-Triage%20AND%20created%20%3E%3D%20-7d%20AND%20affectedVersion%20%3D%20%22OpenShift%204.12%22%20ORDER%20BY%20key%20DESC"
```

Comment:
I think it's not readable or maintainable all this quoted text
Let's do:
```python
import urllib.parse
...
ocp_4_12 = "https://issues.redhat.com/issues/?jql=" + urllib.parse.quote("project = AI-Triage AND component = Cloud-Triage AND create ...")
```

Reply:
Done